hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,16 @@
+RELEASE_TYPE: minor
+
+This release changes the stateful testing backend from
+:func:`~hypothesis.find` to use :func:`@given <hypothesis.given>`
+(:issue:`1300`).  This doesn't change how you create stateful tests,
+but does make them run more like other Hypothesis tests:
+
+- The ``timeout`` setting now applies to stateful tests (:issue:`1213`)
+- Health checks also now apply to stateful tests, so you may need fix
+  them or use :attr:`~hypothesis.settings.suppress_health_check`.
+- :func:`@seed <hypothesis.seed>` and
+  :func:`@reproduce_failure <hypothesis.reproduce_failure>` now work
+  for stateful tests
+
+All tests using :func:`@given <hypothesis.given>` now shrink errors
```

Comment:
Multiple user-facing feature klaxon.

Reply:
OK, fair enough.  I found this when the regression test for #1372 failed under the new backend, but I'll split it out into a separate patch.

Reply:
Done as #1653; I'll return to this PR after that one is merged.